feature/bgp/admin_distance/README.md

```diff
@@ -0,0 +1,94 @@
+*   Verify that the traffic is received on port-1 of the ATE
+
+### RT-1.34.2 [TODO:https://github.com/openconfig/featureprofiles/issues/3050]
+#### Validate traffic with modified eBGP Route-Distance
+*   Configure Route-Distance of eBGP session on port-1 to 220
+    *   /network-instances/network-instance/protocols/protocol/bgp/global/default-route-distance/config/external-route-distance 
+*   Validate using gNMI Subscribe with mode 'ONCE' that the correct Route-Distance value of 220 is reported:
+    *   /network-instances/network-instance/protocols/protocol/bgp/global/default-route-distance/state/external-route-distance
+*   Generate traffic from ATE port-3 towards ```ipv4-network-1 = 192.168.10.0/24``` and ```ipv6-network-1 = 2024:db8:64:64::/64```
+*   Verify that the traffic is received on port-2 of the ATE
+
+### RT-1.34.3 [TODO:https://github.com/openconfig/featureprofiles/issues/3050]
+#### Validate traffic with default Route-Distance of iBGP
+*   Replace IPv4 and IPv6 eBGP with IPv4 and IPv6 iBGP between DUT Port-1 and ATE Port-1
+    *   /network-instances/network-instance/protocols/protocol/bgp/global/config
+    *   /network-instances/network-instance/protocols/protocol/bgp/global/afi-safis/afi-safi/config/
+    *   Advertise ```ipv4-network-1 = 192.168.10.0/24``` and ```ipv6-network-1 = 2024:db8:64:64::/64``` from ATE to DUT over the IPv4 and IPv6 iBGP session on port-1
+*   Generate traffic from ATE port-3 towards ```ipv4-network-1 = 192.168.10.0/24``` and ```ipv6-network-1 = 2024:db8:64:64::/64```
+*   Validate that the traffic is received on port-2 of the ATE
+
+### RT-1.34.3 [TODO:https://github.com/openconfig/featureprofiles/issues/3050]
+#### Validate traffic with modified iBGP Route-Distance
+*   Configure Route-Distance of iBGP session on port-1 to 40
+    *   /network-instances/network-instance/protocols/protocol/bgp/global/default-route-distance/config/internal-route-distance
+*   Validate using gNMI Subscribe with mode 'ONCE' that the correct Route-Distance value of 40 is reported:
+    *   /network-instances/network-instance/protocols/protocol/bgp/global/default-route-distance/state/internal-route-distance
+*   Generate traffic from ATE port-3 towards ```ipv4-network-1 = 192.168.10.0/24``` and ```ipv6-network-1 = 2024:db8:64:64::/64```
+*   Validate that the traffic is received on port-1 of the ATE
+
+## Config Parameter Coverage
+
+* /network-instances/network-instance/protocols/protocol/bgp/global/default-route-distance/config/external-route-distance
+* /network-instances/network-instance/protocols/protocol/bgp/global/default-route-distance/config/internal-route-distance
+
+## Telemetry Parameter Coverage
+
+* /network-instances/network-instance/protocols/protocol/bgp/global/default-route-distance/state/external-route-distance
+* /network-instances/network-instance/protocols/protocol/bgp/global/default-route-distance/state/internal-route-distance
```

Comment:
you can remove this as it it captured in the yaml section below.

Reply:
Ack